.travis.yml

```diff
@@ -1,8 +1,9 @@
 linux: &linux
    os: linux
+   dist: xenial
    sudo: required
```

Comment:
Please remove the sudo when applying the other changes. Travis is deprecating it.

Reply:
@Croydon  is it no longer required for docker service?

Reply:
If the docker service is getting used Travis is implicit setting sudo: true no matter what. And sudo disable environments are soon not available anymore. So let's remove it.